UI/data/locale/en-US.ini

```diff
@@ -845,3 +845,8 @@ OutputWarnings.MP4Recording="Warning: Recordings saved to MP4 will be unrecovera
 # deleting final scene
 FinalScene.Title="Delete Scene"
 FinalScene.Text="There needs to be at least one scene."
+
+# Dowstream Keyer
+DownstreamKeyerScene="Set as Downstream Keyer"
+DownstreamKeyerReset="Reset Downstream Keyer"
+DownstreamKeyerExclude="Exclude from downstream keyer"
```

Comment:
Everywhere else has "Downstream Keyer" capitalized as such.  Is there a particular reason to lowercase it here?